__tests__/app.test.js

```diff
@@ -42,15 +41,49 @@ describe('GET requests', () => {
 
     describe("GET /api", () => {
         test("Status: 200 - Should respond with a JSON object that describes all of the availlable endpoints", () => {
-          return request(app)
-          .get("/api")
-          .expect(200)
-          .then(({body}) => {
-            const {endpoints} = body;
-            expect(typeof endpoints).toBe('object')
-            expect(Object.entries(endpoints).length).toBe(Object.entries(endpointsCopy).length)
+            return request(app)
+            .get("/api")
```

Comment:
Please try not to let other code spill into other branches

Reply:
I'm not sure what you mean.

Reply:
Oh actually. I do. Not sure What happened there. I had lots of git hubbery yesterday